mona_sdk/client.py

```diff
@@ -323,7 +350,7 @@ def _create_client_response(
         }
 
     @Decorators.refresh_token_if_needed
-    def upload_config(self, config, commit_message):
+    def upload_config(self, config, commit_message, author):
```

Comment:
There is no reason in starting to require everyone to use author for upload_config. I'd make this an optional param, and if we have the api_key use that instead, only throw exception if both don't exist.
Please update the readme as well if you make this change...